api/pkg/user/service/user.go

```diff
@@ -250,3 +252,33 @@ func (r *request) newRefreshToken() (*userApp.NewRefreshTokenResult, error) {
 
 	return &userApp.NewRefreshTokenResult{Data: data}, nil
 }
+
+func ParseUserId(str string) (float64, error) {
+	val, err := strconv.ParseFloat(str, 64)
+	if err == nil {
```

Comment:
can we add unit tests for this? Also can we rename this function to some generic name as this can be reused in future?